packages/x-components/src/x-modules/history-queries/components/history-query.vue

```diff
@@ -105,18 +129,47 @@ the suggestion itself, and a `string` of HTML with the matched query.
 The other slot is called `remove-button-content`, and allows you to set the content of the button
 that serves to remove this query from the history. This slot only has one property, the suggestion.
 
-````vue
-<HistoryQuery :suggestion="historyQuery">
-  <template #default="{ suggestion, queryHTML }">
-    <img class="x-history-query__history-icon" src="./history-icon.svg"/>
-    <span class="x-history-query__matching-part" v-html="queryHTML"/>
-  </template>
-
-  <template #remove-button-content="{ suggestion }">
-    <img class="x-history-query__remove-icon" src="./remove-icon.svg"/>
-  </template>
-</HistoryQuery>
-``` ## Events A list of events that the component will emit: - `UserSelectedAHistoryQuery`: the
-event is emitted after the user clicks the button. The event payload is the history query data.
-````
+```vue live
+<template>
```

Comment:
`<template>` can only have one root child. Wrap the components in a div for the code to work